nodejs-server/app.js

```diff
@@ -13,12 +13,12 @@ app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'pug');
 
 app.use(logger('dev'));
-app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
-app.use(cookieParser());
+const payloadSizeLimit = '50mb';
```

Comment:
is this the maximum file size that we could send?

Reply:
Correct-o and I believe there's no limit in wha we put, I think, 50mb for a 1080p-ish screen at one picture at a time is plenty

Wha we could do is combine photos per request like send two at a time

Reply:
so you are not planning to send videos?

Reply:
V2, maybe haha bc i don't have speakers